lib/postal/smtp_server/workers_pool.rb

```diff
@@ -0,0 +1,34 @@
+module Postal
+  module SMTPServer
+    # Workers pool
+    class WorkersPool
```

Comment:
This isn't used anymore, right?

Reply:
Yes, I'll remove it in a final commit, as well as commented old code.